### PR TITLE
Update Readme: Typescript Client

### PR DESCRIPTION
**Pull request by VladislavsPerkanuks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,42 +80,34 @@ docker run -d --name translate-all-in-one \
...
+### Dependencies
 
 ```bash
 npm config set @buf:registry  https://buf.build/gen/npm/v1/
 
-npm install @buf/expectdigital_translate-agent.bufbuild_connect-es@latest
-npm install @buf/expectdigital_translate-agent.bufbuild_es@latest
+npm install @connectrpc/connect
+npm install @connectrpc/connect-web  
+npm install @buf/expectdigital_translate-agent.connectrpc_es@latest
```
Comment (janishorsts) [inline README.md:90]: Not required. These packages are already dependencies.

Commit package.json and pacakge-lock.json instead.

```suggestion
```

Comment (janishorsts) [reply README.md:90]: @egijasp do you use yarn or npm? `yarn.lock` is part of the repo.

File: README.md [hunk 2]
```diff
@@ -80,42 +80,34 @@ docker run -d --name translate-all-in-one \
...
-});
+let tr = new Translation({ language: "en", });
+let req = new CreateTranslationRequest({ translation: tr, });
 
-const client = createPromiseClient(TranslateService, transport);
+client.createTranslation(req).then((res) => {
+  console.log(res);
+}).catch((err) => {
+  console.error(err);
+});
```
Comment (janishorsts) [inline README.md:111]:
```suggestion
client.createTranslation(req)
  .then((res) => console.log(res))
  .catch((err) => console.error(err));
```

File: README.md (hunk 1 above)
Comment (janishorsts) [reply README.md:90]: Ignore my comment! I thought the change is in the webapp.

Comment (egijasp) [reply README.md:90]: at the start, i used yarn, then had problems installing dependencies for grpc client, so I had to use npm.

**Pull request merged**